ui/raidboss/data/02-arr/raid/t11.txt

```diff
@@ -4,20 +4,20 @@
 hideall "--Reset--"
 hideall "--sync--"
 
-0.0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
```

Comment:
I think maybe if we have `hideall "--Reset--"` that these should stay capitalized `--Reset--`?

(Also minor preference for always having a single decimal place on the starting number for consistency?)